swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -286,6 +292,7 @@ let default_translations = {
         created: "Luotu",
         folderDetails: "Ei yksityiskohtia kansioille",
         clearChecked: "Poista valinnat",
+        showTags: "Näytä tagit",
```

Comment:
Probably better as "tägit" with the umlaut – the anglicism is a bit informal/colloquial, but is probably more recognizable for the user

Reply:
Also on the field label that says Tagit, or only here?

Reply:
Preferably change that to "Tägit" as well